.github/workflows/main.yml

```diff
@@ -209,6 +211,62 @@ jobs:
         echo $STAT | jq '.FailedTests' || true
         echo "-------------------------------------------------------"
         if [ "$numFail" -gt 0 ];then echo "*** $numFail Failed ***";exit 2;fi
+  docker_windows_NT:
+    needs: [build_minikube]
+    env:
+      TIME_ELAPSED: time
+      JOB_NAME: "Docker_windows_NT"
+      COMMIT_STATUS: ""
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Download gopogh
+      run: |
+        curl -LO https://github.com/medyagh/gopogh/releases/download/v0.1.16/gopogh.exe
+      shell: bash
+    - name: Download binaries
+      uses: actions/download-artifact@v1
+      with:
+        name: minikube_binaries
+    - name: run integration test
+      continue-on-error: true
+      run: |
+        set +euo pipefail
+        mkdir -p report
+        mkdir -p testhome
+        START_TIME=$(date -u +%s)
+        KUBECONFIG=$(pwd)/testhome/kubeconfig MINIKUBE_HOME=$(pwd)/testhome minikube_binaries/e2e-windows-amd64.exe --expected-default-driver=hyperv -minikube-start-args=--vm-driver=hyperv -binary=minikube_binaries/minikube-windows-amd64.exe -test.v -test.timeout=65m 2>&1 | tee ./report/testout.txt
+        END_TIME=$(date -u +%s)
+        TIME_ELAPSED=$(($END_TIME-$START_TIME)) 
+        min=$((${TIME_ELAPSED}/60))
+        sec=$((${TIME_ELAPSED}%60))
+        TIME_ELAPSED="${min} min $sec seconds"
+        echo ::set-env name=TIME_ELAPSED::${TIME_ELAPSED}
+      shell: bash
+    - name: Generate html report
+      run: |
+        go tool test2json -t < ./report/testout.txt > ./report/testout.json || true
+        STAT=$(${GITHUB_WORKSPACE}/gopogh.exe -in ./report/testout.json -out ./report/testout.html -name " $GITHUB_REF" -repo "${JOB_NAME} ${GITHUB_REF} ${GITHUB_REPOSITORY}"  -details "${GITHUB_SHA}")  || true
+        echo status: ${STAT}
+        FailNum=$(echo $STAT | jq '.NumberOfFail')
+        TestsNum=$(echo $STAT | jq '.NumberOfTests')       
+        GOPOGH_RESULT="${JOB_NAME} : completed with ${FailNum} / ${TestsNum} failures in ${TIME_ELAPSED}"
+        echo ::set-env name=GOPOGH_RESULT::${GOPOGH_RESULT}
+        echo ::set-env name=STAT::${STAT}
+      shell: bash
+    - uses: actions/upload-artifact@v1
+      with:
+        name: docker_windows_NT
+        path: report
+    - name: The End Result
+      run: |
+        echo ${GOPOGH_RESULT}
+        numFail=$(echo $STAT | jq '.NumberOfFail')
+        echo "----------------${numFail} Failures----------------------------"
+        echo $STAT | jq '.FailedTests' || true
+        echo "--------------------------------------------"
```

Comment:
since this PR we added ***Pass
please copy that part from other tests so it inludes the Passes Num